src/infrastructure/figma/figma-service.ts

```diff
@@ -353,7 +354,8 @@ export class FigmaService {
 			if (
 				e instanceof MissingOrInvalidCredentialsFigmaAuthServiceError ||
 				e instanceof UnauthorizedHttpClientError ||
-				e instanceof ForbiddenHttpClientError
+				e instanceof ForbiddenHttpClientError ||
+				e instanceof BadRequestHttpClientError
```

Comment:
**issue:** It can be risky to treat any `HTTP 400 Bad Request` as `UnauthorizedFigmaServiceError`. Instead, we should apply this translation only to specific endpoints (where `HTTP 400 Bad Request` really means "Unauthorized"). I will address this issue.